docker/ci/docker-entrypoint.sh

```diff
@@ -62,6 +62,10 @@ function run_test ()
   CTP/bin/ini.sh -s sql/cubrid.conf CTP/conf/medium.conf create_table_reuseoid no
   cd -
 
+  cd $WORKDIR/cubrid-testtools/CTP
+  ant clean dist
```

Comment:
```suggestion
  ant dist
```
dist has clean dependency

Reply:
OK